pkg/kubelet/logs/container_log_manager.go

```diff
@@ -177,26 +179,23 @@ func (c *containerLogManager) Start() {
 }
 
 func (c *containerLogManager) rotateLogs() error {
-	// TODO(#59998): Use kubelet pod cache.
-	containers, err := c.runtimeService.ListContainers(&runtimeapi.ContainerFilter{})
-	if err != nil {
-		return fmt.Errorf("failed to list containers: %v", err)
+	// get PodStatus from kubelet pod cache
```

Comment:
Nit: I think this comment is unnecessary - the variable names in the line below make it pretty clear what's happening.